### PR TITLE
Add a convenience IO header

### PR DESCRIPTION
**Pull request by MaelRL**
## Summary of Changes

There is:

```
CGAL/boost/graph/IO/polygon_mesh_io.h
CGAL/PMP/IO/polygon_mesh_io.h
CGAL/IO/polygon_soup_io.h
```

The first one is a bit awkward to remember, so this PR adds a convenience header at `CGAL/IO/polygon_mesh_io.h`.

## Release Management

* Affected package(s): `BGL`
* Issue(s) solved (if any): -
* Feature/Small Feature (if any):
* Link to compiled documentation (obligatory for small feature) [*wrong link name to be changed*](httpssss://wrong_URL_to_be_changed/Manual/Pkg)
* License and copyright ownership:

### CODE REVIEW

File: BGL/include/CGAL/IO/polygon_mesh_io.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Mael Rouxel-Labbé
+
+#ifndef CGAL_BOOST_GRAPH_POLYGON_MESH_IO_H
+#define CGAL_BOOST_GRAPH_POLYGON_MESH_IO_H
```
Comment (sloriot) [inline BGL/include/CGAL/IO/polygon_mesh_io.h:12]:
```suggestion
#ifndef CGAL_IO_POLYGON_MESH_IO_H
#define CGAL_IO_POLYGON_MESH_IO_H
```

File: BGL/include/CGAL/IO/polygon_mesh_io.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+#include <CGAL/boost/graph/io.h>
+
+#endif // CGAL_BOOST_GRAPH_POLYGON_MESH_IO_H
```
Comment (sloriot) [inline BGL/include/CGAL/IO/polygon_mesh_io.h:16]:
```suggestion
#endif // CGAL_IO_POLYGON_MESH_IO_H
```

**Comment by lrineau** [comment]
`CGAL/boost/graph/IO/polygon_mesh_io.h` is still the documented header in the following doc pages:

```
BGL/group__PkgBGLIOFct.html
BGL/index.html
Polygon_mesh_processing/index.html
Shape_detection/index.html
Straight_skeleton_2/index.html
Surface_mesh_simplification/index.html
```

Dont' you want to change that as well, in addition to the changes in examples?

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-129

**Pull request merged**